src/usr/local/emhttp/plugins/tailscale/include/Pages/Log.php

```
@@ -12,7 +12,13 @@
 ?>
 <?php
 
-$tr = $tr ?? new Tailscale\Translator();
+use EDACerton\PluginUtils\Translator;
+
+if ( ! defined('Tailscale\PLUGIN_ROOT') || ! defined('Tailscale\PLUGIN_NAME')) {
+    throw new \RuntimeException("Common file not loaded.");
+}
+
+$tr = $tr ?? new Translator(Tailscale\PLUGIN_ROOT);
```

PHPStan (2.1.15)

Instantiated class EDACerton\PluginUtils\Translator not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)